### DISCUSSION TITLE
Error after installing all dependencies

### DISCUSSION THREAD

**Discussion by ErikDz** [pymupdf/PyMuPDF#857] [Looking for help] [answered]
Hi, I get the following error upon importing fitz after having had imported all dependencies:

> Traceback (most recent call last):
>   File "c:\Users\directora\Documents\PityPython\PastPaperFilterV3\question_grabber.py", line 1, in <module>
>     import fitz
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\fitz\__init__.py", line 1, in <module>  
>     from frontend import *
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\frontend\__init__.py", line 1, in <module>
>     from .events import *
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\frontend\events\__init__.py", line 1, in <module>
>     from .clipboard import *
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\frontend\events\clipboard.py", line 2, in <module>
>     from ..dom import Event
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\frontend\dom.py", line 439, in <module> 
>     from . import dispatcher
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\frontend\dispatcher.py", line 15, in <module>
>     from . import config, server
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\frontend\server.py", line 24, in <module>
>     app.mount(config.STATIC_ROUTE, StaticFiles(directory=config.STATIC_DIRECTORY), name=config.STATIC_NAME)
>   File "C:\Users\directora\AppData\Local\Programs\Python\Python38-32\lib\site-packages\starlette\staticfiles.py", line 57, in __init__
>     raise RuntimeError(f"Directory '{directory}' does not exist")
> RuntimeError: Directory 'static/' does not exist

**Comment by JorjMcKie** [comment, marked as answer]
I think you are one of those who installed a package named ``fitz``.
This is **_not_** PyMuPDF - but something entirely different. Because it has the same top-level text ``fitz`` it cannot coexist with PyMuPDF in the same environment.

**Reply by ErikDz** [reply]
Oh silly me!
Apologies, I should have checked
Thank you so much for the answer.
